xbmc/pvr/PVRDatabase.cpp

```diff
@@ -143,6 +143,9 @@ void CPVRDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 30)
     m_pDS->exec("ALTER TABLE channels ADD bWasPlayingOnQuit bool");
+
+  if (iVersion < 31)
+    m_pDS->exec("ALTER TABLE channels DROP bWasPlayingOnQuit");
```

Comment:
There is no support for dropping columns in SQL lite. Sorry for that, I thought it is. Kodi will report the following error. We'll provide a fix.

```
10:02:07.370 T:140472773244672  NOTICE: Old database found - updating from version 30 to 31
10:02:07.382 T:140472773244672  NOTICE: Attempting to update the database TV31 from version 30 to 31
10:02:07.383 T:140472773244672   ERROR: SQL: [TV31.db] SQL error or missing database
                                            Query: ALTER TABLE channels DROP bWasPlayingOnQuit
10:02:07.383 T:140472773244672   ERROR: Exception updating database TV31 from version 30 to 31
10:02:07.383 T:140472773244672   ERROR: Error updating database TV31 from version 30 to 31
```